packages/ariakit/src/button/__examples__/button-as-div/style.css

```
@@ -0,0 +1,21 @@
+.button {
+  @apply flex
+    items-center
+    justify-center
+    gap-2
+    text-base
+    cursor-pointer
+    border-none
+    rounded-lg
+    px-4
+    h-10
+    bg-primary-2
+    text-primary-2
+    hover:bg-primary-2-hover
+    dark:bg-primary-2-dark
+    dark:text-primary-2-dark
+    dark:hover:bg-primary-2-dark-hover
+    ariakit-focus-visible:ariakit-outline
+    aria-disabled:opacity-50
+  }
+  
```

I think you can just import the default styles in this case.
```suggestion
@import url("../button/style.css");

```

---

Oh, yeah, this is good 👍

---

Hey @diegohaz , take a look at this, made the change